install/kubernetes/templates/helm/istio/values.yaml.tmpl

```diff
@@ -3,8 +3,8 @@ Istio:
    initializer_enabled: true 
    mixer_enabled: true 
    pilot_enabled: true 
```

Comment:
I think we need to re-evaluate the need for 5 hubs and tags.

There are no plans to host releases in different hubs, and for now we have a single release. Advanced users who build their own images can customize ( in particular if they use multiple repositories).